apps/sms/js/thread_list_ui.js

```diff
@@ -646,7 +648,7 @@ var ThreadListUI = {
     Settings.setReadAheadThreadRetrieval(this.FIRST_PANEL_THREAD_COUNT);
   },
 
-  renderThreads: function thlui_renderThreads(firstViewDoneCb, allDoneCb) {
+  renderThreads: function thlui_renderThreads(firstViewDoneCb) {
```

Comment:
maybe not to do now, but how would you feel about returning an object with 2 promises ?